### DISCUSSION TITLE
cheat_list.MD improvement

### DISCUSSION THREAD

**Discussion by Slider2k** [PS2-Widescreen/OPL-Widescreen-Cheats#57] [Ideas]
Could you please add direct links to CHT files in [cheat_list.MD](https://github.com/PS2-Widescreen/OPL-Widescreen-Cheats/blob/7ae7e2f2ff66b2fa4b11670bc2ca2214d07351ec/cheat_list.MD)? Would greatly help in navigation!

**Comment by israpps** [comment]
> Could you please add direct links to CHT files in [cheat_list.MD](https://github.com/PS2-Widescreen/OPL-Widescreen-Cheats/blob/7ae7e2f2ff66b2fa4b11670bc2ca2214d07351ec/cheat_list.MD)? Would greatly help in navigation!

Done!

https://github.com/PS2-Widescreen/OPL-Widescreen-Cheats/blob/main/cheat_list.MD
